### DISCUSSION TITLE
Suggestion: Link Graveyards and links to graveyards to openbenches

### DISCUSSION THREAD

**Discussion by backache** [openbenches/openbenches.org#320] [Ideas]
As Graveyards and crematoria are likely places where clusters of memorial benches will be, should there be a way to link them, for example, "Croydon Crematorium" would currently just be its geographic coordinates

openbenches.org/#51.39244/-0.1233602/16

However, there are few databases out there that log data about these sites (and the people at rest there) where interlinking could be mutually beneficial, for example for ones listed in croydon's wikidata entry https://www.wikidata.org/wiki/Q5189647 are -

BillionGraves cemetery ID
CWGC burial ground ID
Find A Grave cemetery ID

as well as the more generic interlinking

Freebase ID
Google Maps CID

**Comment by edent** [comment]
We already have this with the geolocation finder.

For example - https://openbenches.org/location/UK,%20Weston-Super%20Mare,%20Ebdon%20Road%20Cemetery - that'll show you all the benches in that graveyard.

I like the idea of integrating WikiData. What would expect the user interface to look like?
